### ISSUE TITLE
Base map

### ISSUE THREAD

**Issue by JanaMynarova** [willismonroe/anax#13]
Hi Willis, I apologize if this is my fault, but in the older version the base map was without modern names, borders, etc. Today I wanted to create a new map, but I don't know how to go back to the original base. This one doesn't suit me because the amount of modern and ancient places creates crazy confusion. Can you give me some advice on how to do this? Thanks.

**Comment by willismonroe** [comment]
Hi Jana, there's something going on with the University of Iowa's map server (where we get the non-modern tiles from), it seems to work for me now I just have to let the map sit for 5 seconds or so.

Let me know if you're still having problems.

**Comment by JanaMynarova** [comment]
Hi Willis, sorry to be such a burden but I have tried Safari, Chrome, and Firefox and nothing works. I can still see all the modern details, no matter how long I wait :(
<img width="1440" alt="Snímek obrazovky 2023-08-09 v 21 15 39" src="https://github.com/willismonroe/anax/assets/141831656/072452e2-b997-4611-9624-62db809b975a">

**Comment by willismonroe** [comment]
Yeah, ok, I think I know what's going on. When I was first looking into this I visited: https://cawm.lib.uiowa.edu/#tiles/ol3 and saw that their certificate was expired. So I did the whole "this is unsafe" etc... "proceed anyway" and then I was able to see their site... that choice was probably saved in my browser and now when the anax script calls for those map tiles it knows that I don't mind that their certificate is expired. So you could try that.

I've also e-mailed folks to figure out how to get the UIowa library to update their certificate... hopefully we can sort that out soon.

**Comment by JanaMynarova** [comment]
Amazing! Thanks a lot. Problem solved (for the moment) but I hope they will update the certificate soon.

**Comment by JanaMynarova** [comment]
Sorry to bother you again. The map is back to normal but it seems that I cannot add the sites to it. Any advice?

**Comment by willismonroe** [comment]
Glad to hear it! I also added a toggle for grayscale ad color today...

**Comment by willismonroe** [comment]
Oh you're right... I'll work on that right away.

**Comment by JanaMynarova** [comment]
Thank you so much!

**Comment by willismonroe** [comment]
Ah it's because github is down for some reason... so luckily it's not my fault, but it does prevent the tool from working. I'll keep trying at it and post a reply once it's back up.

**Comment by willismonroe** [comment]
Just as I sent that it came back up, should be working now.
